src/main/kotlin/it/pagopa/helpdeskcommands/services/CommandsService.kt

```diff
@@ -36,7 +36,7 @@ class CommandsService(
                 logger.info(
                     "Performing NPG refund for transaction with id: [{}] and paymentMethod: [{}]. " +
                         "OperationId: [{}], amount: [{}], pspId: [{}], correlationId: [{}]",
-                    transactionId,
+                    transactionId.uuid,
```

Comment:
```suggestion
                    transactionId.value,
```

In log just use transaction id without dashes, since it's the id of the transaction as stored into eCommerce.
I would also add a separate logging entry for idempotency key logging the `transactionId.uuid` value